general-concepts/use-flags/text.xml

```diff
@@ -276,14 +276,14 @@ src_compile() {
 	elif use ssl &amp;&amp; ! use gnutls ; then
 		myconf="${myconf} --enable-ssl --with-ssl=openssl"
 	else
-	myconf="${myconf} --disable-ssl"
+		myconf="${myconf} --disable-ssl"
```

Comment:
All these assignments could profit from using +=, e.g. `myconf+=" --disable-ssl"`

Reply:
Why `+=`? There is no previous assignment in this example.
I think the example should be rewritten if we want to show profit from using `+=`. But that is for another PR.

Reply:
Sure, we can be strict and split issues into different PRs. But then that indentation fix doesn't belong here either. :stuck_out_tongue:

Reply:
Sure:D